### ISSUE TITLE
Investigate solution for g8

### ISSUE THREAD

**Issue by renatocaval** [lightbend/templatecontrol#62]
Template control is to intrusive with g8 templates. It removes the g8 variables. 

We should investigate the possibility to exclude some replacements for g8 templates.

**Comment by renatocaval** [comment]
I think the easiest way to solve it is to check if there is anything in the matched line that resembles a g8 variable and then ignore the line. 

Other possible solution is to add a command at the end. 
```
def playVersion = '$play_version$'  // tc-skip
```
Any line marked with `tc-skip` should be skipped. WDYT?

**Comment by marcospereira** [comment]
Ideally, the seeds shouldn't need any substitutions. To get versions, for example, we should be able to use `stable` (see https://github.com/foundweekends/giter8/pull/275):

```properties
play_version=maven(com.typesafe.play, play_2.12, stable)
```

But as far as I remember, this does not work for sbt prior 1.2.0 (didn't test with 1.2.x though). Then when we released 2.7.0-M1, using only `maven(com.typesafe.play,play_2.12)` was actually returning M1 and we moved to hardcoded versions (https://github.com/playframework/play-scala-seed.g8/pull/53).

Long story short, after releasing a stable version of 2.7.x, we can use `maven(com.typesafe.play,play_2.12)` and ensure users are getting the latest release without modifying the seeds.

**Comment by marcospereira** [comment]
Another option is not to use the giter8 variables, but instead, hardcode the versions where they are needed. It is what Lagom is doing, and we don't have a strong reason to not follow in Play. :-)

**Comment by renatocaval** [comment]
@marcospereira I had the code ready already and did some tests today, so I'm sending the PR anyway. 

We can discuss and decide to drop it eventually. 

I agree that for a template, we can stick with the stable version. If someone wants to start a new project using a previous version, they can change it manually. 

Maybe, we should simply not use variables at all and let template control update it accordingly. In that case, we don't need to use the stable release trick which means that we don't need to care about milestone releases. As soon as we have a new release we run template control and everything gets updated.
